pump/storage/storage.go

```diff
@@ -651,7 +650,9 @@ func (a *Append) GC(ts int64) {
 }
 
 func (a *Append) doGCTS(ts int64) {
-	log.Info("start gc", zap.Int64("ts", ts))
+	log.Info("Starting GC", zap.Int64("ts", ts))
+
+	a.vlog.gcTS(ts)
```

Comment:
May I ask a question here? If we `gcTS(ts)` here, why we need to do `a.vlog.gcTS(decodeTSKey(lastKey))`? Because the storage is concurrently adding vlogs?

Reply:
yes, we store metadata( ts -> (vlogfile,offset) ) in goleveldb, and store binlog data in vlog.

Reply:
@lichunzhu Good observation! I think we can get rid of `a.vlog.gcTS(decodeTSKey(lastKey))` if we are doing the whole `gcTS` thing concurrently.